### PR TITLE
update scenarios

### LINKED ISSUE #17: Add a list of all scenarios into features/README.md file
Add a list of all scenarios into features/README.md file

### PR DESCRIPTION
**Pull request by juandspy**
# Description

There were some scenarios missing due to the unification we did some weeks ago.

Fixes #17

## Type of change

- Documentation update

## Checklist
* [x] Pylint passes for Python sources
* [x] sources has been pre-processed by Black
* [x] updated documentation wherever necessary

### CODE REVIEW

File: tools/gen_scenario_list.py [hunk 1]
```diff
@@ -28,15 +28,18 @@
...
-SUBDIRECTORIES = ("OCM",
-                  "OCP_WebConsole",
-                  "ACM",
-                  "Insights_Advisor",
-                  "Notification_Service",
-                  "SHA_Extractor",
-                  "insights-results-aggregator-cleaner",
-                  "insights-results-aggregator-exporter",
-                  )
+SUBDIRECTORIES = (
```
Comment (tisnik) [inline tools/gen_scenario_list.py:31]: if you don't want to maintain original order (and make harder to see history of changes in README.md) this is not needed

Comment (juandspy) [reply tools/gen_scenario_list.py:31]: do we want to maintain it?

**Review by epapbak — APPROVED** [review]
thanks =)

**Review by tisnik — APPROVED** [review]
ok TY

**Pull request merged**